libs/template/templates/dbt-sql/template/{{.project_name}}/src/models/example/schema.yml

```diff
@@ -16,6 +16,6 @@ models:
     columns:
       - name: order_date
         description: "The date on which orders took place"
-        tests:
+        data_tests:
```

Comment:
This would require dbt 1.8.0, which is just fine, since we basically already require that for materialized views and streaming tables.

Could you update https://github.com/databricks/cli/blob/main/libs/template/templates/dbt-sql/template/%7B%7B.project_name%7D%7D/resources/%7B%7B.project_name%7D%7D_job.yml.tmpl#L38 to say 1.8.0 accordingly?

Reply:
I have amended previous commit and added dbt 1.8.0 to `{{.project_name}}_job.yml.tmpl` and `requirements-dev.txt` files

Reply:
Thanks! That looks great